content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -103,6 +103,10 @@ The following methods exist for installing kubectl on macOS:
    sudo chown root: /usr/local/bin/kubectl
    ```
 
+   {{< note >}}
+   On macOS,  `/usr/local/bin/` is usually included in your `$PATH`.
```

Comment:
How about "Make sure `/usr/local/bin` is in your PATH environment variable."

Reply:
That pretty good👍